test/unit/compiler/test-builders.js

```diff
@@ -52,5 +56,27 @@ describes.fakeWin('getBuilders', {}, (env) => {
       builders.noop(elem);
       expect(elem.getAttribute('i-amphtml-layout')).equal('fixed');
     });
+
+    it('wrapper should behave same in browser as in WorkerDOM', () => {
+      const browserDiv = createElementWithAttributes(doc, 'div', {
+        height: 100,
+        width: 100,
+      });
+      const workerDomDiv = createElementWithAttributes(
+        createWorkerDomDoc(),
+        'div',
+        {
+          height: 100,
+          width: 100,
+        }
+      );
+
+      builders.noop(browserDiv);
+      builders.noop(workerDomDiv);
+
+      const browserHtml = getDeterministicOuterHTML(browserDiv);
+      const workerHtml = getDeterministicOuterHTML(workerDomDiv);
+      expect(workerHtml).equal(browserHtml);
```

Comment:
This is a super common pattern with some typo potential. Might be more useful to directly export `expectDeterministicOuterHtmlEqual(worker, browser)`

Reply:
Considered this! Don't love the pattern of making helpers that perform assertions. I know when I read it in other folk's code it always takes me longer to understand the test case.

I have a preference for under-abtracting in unit tests (keeping them [DAMP](https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests)). That said, especially in this case, don't have a very strong opinion